### ISSUE TITLE
Build on Red Hat

### ISSUE THREAD

**Issue by haonans** [tensorflow/serving#145]
I am trying to build serving on red hat, here is the error I encoutered:

```
$ bazel build tensorflow_serving/...
WARNING: Sandboxed execution is not supported on your system and thus hermeticity of actions cannot be guaranteed. See http://bazel.io/docs/bazel-user-manual.html#sandboxing for more information. You can turn off this warning via --ignore_unsupported_sandboxing.
ERROR: /home/haonans/serving/tensorflow/tensorflow/workspace.bzl:98:3: no such package '@gmock_archive//': Error downloading from https://archive.openswitch.net/gmock-1.7.0.zip to /home/haonans/.cache/bazel/_bazel_haonans/c4b92d0b49abc3b86029056d06434013/external/gmock_archive: Error downloading https://archive.openswitch.net/gmock-1.7.0.zip to /home/haonans/.cache/bazel/_bazel_haonans/c4b92d0b49abc3b86029056d06434013/external/gmock_archive/gmock-1.7.0.zip: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target and referenced by '//external:gtest'.
ERROR: Analysis of target '//tensorflow_serving/test_util:test_util' failed; build aborted.
```

How to resolve this problem?

Many thanks!

**Comment by jamesatha** [comment]
This looks like an issue with an SSL certificate. I'm also blocked on the same issue. Is there a workaround for this yet? Thanks!

**Comment by moorea** [comment]
Same issue here

**Comment by haonans** [comment]
@moorea @jamesatha

**Comment by haonans** [comment]
@jamesatha @moorea 

Here is the java file you will need. You can just follow this [guide](https://github.com/escline/InstallCert), use "archive.openswitch.net" as the hostname.

When you try to run build bazel from source try:

bazel build --host_jvm_args=-server  tensorflow_serving/...

or 

bazel build --host_jvm_args=-server  tensorflow_serving/... -c opt

And probably you will solve this problem (although actually I don't know how and why I solved this randomly, lol)

**Comment by xuanhua** [comment]
@haonans Hi, haonans, I meet this issue when have a try on 'syntaxnet' and i just tried your suggestions, it just does not work. And currently syntaxnet is based on this version of tensorflow:

commit 861644c0bcae5d56f7b3f439696eefa6df8580ec
Merge: 1d7ba2b a76f93a
Author: Rasmus Munk Larsen <rmlarsen@google.com>
Date:   Mon Jun 27 11:20:14 2016 -0700

    Merge pull request #2985 from sharkdtu/master
    
    fix word2vec_test's tmp path

**Comment by kirilg** [comment]
Closing old issue that's likely outdated. Please file a new issue if this is a problem at head.
